tinycrypt/ecc_dsa.c

```diff
@@ -251,6 +257,7 @@ int uECC_verify(const uint8_t *public_key, const uint8_t *message_hash,
 	uECC_vli_set(ry, point + num_words);
 	uECC_vli_clear(z);
 	z[0] = 1;
+	flow_control++;
 
 	for (i = num_bits - 2; i >= 0; --i) {
```

Comment:
@AndrzejKurek , @piotr-now ,Do you see any reason for this over complicated **"for"** condition ?

Reply:
What do you mean exactly? `uECC_vli_testBit` uses the `i` value.